Fix/pretty name btn (#1341)
kedro-org/kedro-viz: RELEASE.md

- Fix SQLAlchemy QueuePool Overflow Error. (#1301)
- Bug fix for where some elements on the flowchart weren't clickable. (#1322)

> - Bug fix for pretty name config in the settings panel is updated even when clicking on cancel button. (#1341)

Reviewer: ```suggestion
- Bug fix for pretty name option in settings panel being updated even when clicking on cancel button. (#1341)
```
